packages/data-context/src/sources/MigrationDataSource.ts

```diff
@@ -62,6 +62,18 @@ export class MigrationDataSource {
     return this.ctx.coreData.migration.legacyConfigForMigration
   }
 
+  get shouldMigratePreExtension () {
+    return !this.legacyConfig.projectId
+  }
+
+  get migrationOptions () {
```

Comment:
Seems like we have this twice, here and also here? https://github.com/cypress-io/cypress/pull/21193/files#diff-c2c1f114b54072e7f036c272e41454586ddba1378ada7bf61d38049e05e3d8f5R68

Reply:
Also what is the point of this getter? Why not just access `shouldMigratePreExtension` directly?